runtime/testnet/src/config/assets.rs

```diff
@@ -30,6 +30,8 @@ parameter_types! {
 parameter_types! {
 	pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
 	pub const NftsCollectionDeposit: Balance = 10 * UNIT;
+	/// Key = 32 bytes (16+16), Value = 68 bytes (4+32+32)
```

Comment:
My calculation could be wrong so having second eyes on this calculation would help a lot. 
In SCALE codec, the Option takes one byte but in normal Rust type would take up to 16 bytes for `Option<BlockNumber>` due to data alignment (8 bytes for option and 8 bytes for BlockNumber).

Reply:
Resolved with test.